src/pages/MarketPoolsPage/MarketPoolsPage.tsx

```diff
@@ -47,7 +47,7 @@ export function MarketPoolsPage() {
     undefined
   );
 
-  const [selectedGlvGmMarketKey, setSelectedGlvGmMarketKey] = useState<string | undefined>(undefined);
+  const [selectedGlvGmMarket, setSelectedGlvGmMarket] = useState<string | undefined>(undefined);
```

Comment:
glvGmMarket -> market

Reply:
I guess it's better to rename also 

selectedMarketGmKey -> selectedMarketOrGlvKey

but this in `selectedGlvGmMarket` I'd like to keep in naming information about that this market is only selected for GLV's underlying market. THere's differences between selecting market in Market selector and selecting Market in Pool selector for GLV (both behavior and internal logic, like GLV's markets shouldn't keep in localstorage etc) 

What about `selectedMarketForGlv`?

Reply:
Agreed on selectedMarketForGlv. Renamed